README.md

```diff
@@ -1,4 +1,11 @@
-# `latexify`
+# latexify
+[![Python](https://img.shields.io/pypi/pyversions/latexify-py.svg?style=plastic)](https://badge.fury.io/py/latexify-py)
```

Comment:
The fury.io address actually points to the PyPI. We don't need to add additional third-party address.

```suggestion
[![Python](https://img.shields.io/pypi/pyversions/latexify-py.svg)](https://pypi.org/project/latexify-py/)
```